docs/tutorials/06_torch_runtime.ipynb

```diff
@@ -0,0 +1,1621 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate random dataset\n",
+    "\n",
+    "# Select dataset dimension (num_inputs) and size (num_samples)\n",
+    "num_inputs = 2\n",
+    "num_samples = 20\n",
+    "\n",
+    "# Generate random input coordinates (X) and binary labels (y)\n",
+    "X = 2 * algorithm_globals.random.random([num_samples, num_inputs]) - 1\n",
+    "y01 = 1 * (np.sum(X, axis=1) >= 0)  # in { 0,  1}, y01 will be used for CircuitQNN example\n",
+    "y = 2 * y01 - 1  # in {-1, +1}, y will be used for OplowQNN example\n",
+    "\n",
+    "# Convert to torch Tensors\n",
+    "X_ = Tensor(X)\n",
+    "y01_ = Tensor(y01).reshape(len(y)).long()\n",
+    "y_ = Tensor(y).reshape(len(y), 1)\n",
+    "\n",
+    "# Plot dataset\n",
+    "for x, y_target in zip(X, y):\n",
+    "    if y_target == 1:\n",
+    "        plt.plot(x[0], x[1], \"bo\")\n",
+    "    else:\n",
+    "        plt.plot(x[0], x[1], \"go\")\n",
+    "plt.plot([-1, 1], [1, -1], \"--\", color=\"black\")\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create custom torch dataset class\n",
+    "class TorchDataset(Dataset):\n",
+    "    \"\"\"Map-style dataset\"\"\"\n",
+    "\n",
+    "    def __init__(self, X, y):\n",
+    "        self.X = Tensor(X)\n",
+    "        self.y = Tensor(y)\n",
+    "\n",
+    "    def __len__(self):\n",
+    "        return len(self.X)\n",
+    "\n",
+    "    def __getitem__(self, idx):\n",
+    "        import torch\n",
+    "\n",
+    "        if torch.is_tensor(idx):\n",
+    "            idx = idx.tolist()\n",
+    "\n",
+    "        X_i = self.X[idx]\n",
+    "        y_i = self.y[idx]\n",
+    "\n",
+    "        # important: the dataset item must be returned as data,target\n",
+    "        return X_i, y_i\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y = y.reshape(20, 1)\n",
+    "train_set = TorchDataset(X, y)\n",
+    "train_loader = DataLoader(train_set, batch_size=1, shuffle=False)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
```

Comment:
This is just an empty cell that is rendered is as "Type Markdown and LaTeX: 𝛼2" in the notebook. I guess, it may be safely deleted.